tzrec/modules/intervention.py

```diff
@@ -0,0 +1,36 @@
+import torch
+from torch import nn
+
+class RotateLayer(nn.Module):
+    def __init__(self, 
+                base_dim,
+                low_rank_dim):
+        super().__init__()
+        # n > m
```

Comment:
base_dim must greater than low_rank_dim?
```
assert base_dim > low_rank_dim
```

Reply:
Added the assertion